src/main/java/com/parkingapp/parkingservice/domain/parkingzone/ParkingZone.java

```diff
@@ -11,4 +13,7 @@ public class ParkingZone {
     private UUID id;
     private String name;
     private UUID cityId;
+    private Location location;
+    private Currency currency;
```

Comment:
```suggestion
    private CurrencyUnit currency;
```
We could use CurrencyUnit from the javax.money library instead of building our own enum class: https://www.baeldung.com/java-money-and-currency